test/common/router/config_impl_test.cc

```diff
@@ -6692,6 +6693,7 @@ TEST_F(RoutePropertyTest, TestVHostCorsConfig) {
   EXPECT_EQ(cors_policy->exposeHeaders(), "test-expose-headers");
   EXPECT_EQ(cors_policy->maxAge(), "test-max-age");
   EXPECT_EQ(cors_policy->allowCredentials(), true);
+  EXPECT_EQ(cors_policy->allowPrivateNetworkAccess(), true);
```

Comment:
May a simple test to ensure `cors_policy->allowPrivateNetworkAccess()` is `false` when we have no `allow_private_network_access` or `allow_private_network_access` is set to `false`?

Reply:
Add it in cors tests.